spaceapi/src/utils.rs

```diff
@@ -108,5 +152,26 @@ impl<T> Optional<T> {
             _ => false
         }
     }
+}
 
+impl<T> Into<Option<T>> for Optional<T> {
+    /// Convert Optional<T> into Option<T>
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # use spaceapi::utils::Optional;
+    /// # use spaceapi::utils::Optional::{Value,Absent};
+    /// let x: Optional<u32> = Value(2);
+    /// assert_eq!(Some(2), x.into());
+    ///
+    /// let x: Optional<u32> = Absent;
+    /// assert_eq!(None, x.into());
+    /// ```
+    fn into(self) -> Option<T> {
```

Comment:
`into()` is defined by the `Into` trait: https://doc.rust-lang.org/std/convert/trait.Into.html
Rust most of the time deduces the target type, otherwise one can specify it:
```Rust
let y = Optional::Absent;
let x : Option<i32> = y.into();
```